### PR TITLE
Add comments using involvement api

### PR DESCRIPTION
**Pull request by Efiamotu-1**
## Features Implemented

>- use involvement api to fetch users comments
>- comments.css to style the comments tab
>- add commets to popup.js file

### CODE REVIEW

File: src/popup.js [hunk 1]
```diff
@@ -29,7 +36,21 @@ const openModal = async (id) => {
   </div>
       <h3 class="summary">Synopsis</h3>
       <p class="series-desc">${shows[id].summary.replaceAll(/<b>|<\/b>|<p>|<\/p>/g, '')}</p>
+      <div class="comment-section">
+      <h4 class="center-comment">Comments (2)</h4>
```
Comment (mullenlucas) [inline src/popup.js:40]: This line, where it says "Comments (2)" should display number of comments of this id (maybe the length of the array of the comments)

**Review by mullenlucas — CHANGES_REQUESTED** [review]
# STATUS: Pending changes

## Hi @Efiamotu-1   👋, 


Very good job so far !

## Highlights 🎯 
✔️ App working as expected
✔️ Descriptive PR
✔️ Used GitFlow
✔️ No linter errors
✔️ Clean code

The only thing I found that would change is one the popup.js, at line 40, where it says "Comments(2)", kindly change that number to the specific of that id's comments (as shown in the comment above)

Other than that, great work !

File: src/popup.js (hunk 1 above)
Comment (Efiamotu-1) [reply src/popup.js:40]: okay, thank you very much for your review @mullenlucas , however this feature isn't part of the requirement for this task but i will make the change and commit it so you can approve

**Review by mullenlucas — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi @Efiamotu-1   👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎯 
✔️ App working as expected
✔️ Descriptive PR
✔️ Used GitFlow
✔️ No linter errors
✔️ Clean code

Cheers and Happy coding!👏👏👏

**Pull request merged**